src/github.com/matrix-org/go-neb/services/slackapi/slackapi.go

```diff
@@ -38,11 +38,14 @@ func (s *Service) OnReceiveWebhook(w http.ResponseWriter, req *http.Request, cli
 
 	slackMessage, err := getSlackMessage(*req)
 	if err != nil {
+		log.WithFields(log.Fields{"slackMessage":slackMessage, "err":err}).Print("Slack message error")
```

Comment:
Also, you should never specify the error key directly, use `log.ErrorKey` instead.